demisto_sdk/commands/validate/validators/IF_validators/IF117_invalid_marketplaces_in_alias.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import annotations
+
+from typing import Iterable
+
+from packaging.version import Version
+
+from demisto_sdk.commands.common.constants import (
+    OLDEST_INCIDENT_FIELD_SUPPORTED_VERSION,
+    MarketplaceVersions,
+)
+from demisto_sdk.commands.common.logger import logger
+from demisto_sdk.commands.content_graph.common import ContentType
+from demisto_sdk.commands.content_graph.objects.incident_field import IncidentField
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = IncidentField
+
+
+class IsValidAliasMarketplaceValidator(BaseValidator[ContentTypes]):
+    error_code = "IF117"
+    description = "Checks if marketplace value in aliases is valid."
+    rationale = "marketplace in aliases should be ['xsoar']."
+    error_message = (
+        "The following fields exist as aliases and have invalid 'marketplaces' key value: \n{aliases}\n "
+        "the value of the 'marketplaces' key in these fields should be ['xsoar']."
+    )
+    related_field = "Aliases"
+
+    def obtain_invalid_content_items(
+        self, content_items: Iterable[ContentTypes]
+    ) -> list[ValidationResult]:
+        return [
+            ValidationResult(
+                validator=self,
+                message=self.error_message.format(aliases=", ".join(aliases)),
+                content_object=content_item,
+            )
+            for content_item in content_items
+            if (aliases := self.invalid_aliases_marketplace(content_item))
+        ]
+
+    def invalid_aliases_marketplace(self, content_item):
+        """Checks if the marketplace of the aliases is valid.
+        Args:
+            content_item : The content item.
+        Returns:
+            list[str]: A list of the names of aliases that have invalid marketplace.
+        """
+        invalid_aliases = []
+        aliases = content_item.data.get("Aliases", [])
+        if aliases:
+            if not self.graph:
```

Comment:
Please refer to any of the GR validations for reference of how the new validate is working with graph.